pages/app/index.html

```diff
@@ -27,12 +27,19 @@
       if (location.hash.includes('file-upload')) {
         csp['img-src'] = 'blob:';
       }
-
+      
       const cspString = Object.entries(csp)
         .map(([key, value]) => `${key} ${value}`)
         .join('; ');
 
       document.write('<meta http-equiv="Content-Security-Policy" content="' + cspString + '" />');
+      
+      if (location.hash.includes('removeHighContrastHeader')) {
```

Comment:
This script is only for building the CSP configuration, because it is not possible to set it otherwise.

Can we keep this script clean and set it in a more appropriate place, same as we do for visual refresh: https://github.com/cloudscape-design/components/blob/9f65ea408177be13561dc2c2e2f56e90f3b1f351/pages/app/index.tsx#L82-L86

Reply:
Great point, addressed